app/db/models.py

```diff
@@ -99,32 +194,29 @@ class Job(Base):
 
     def __init__(
         self,
-        status: JobStatus,
-        aoi_id: int,
-        model_id: int,
         start_date: datetime.datetime,
         end_date: datetime.datetime,
         maxcc: float,
-        is_deleted: bool = False,
-        created_at: datetime.datetime = datetime.datetime.now(),
+        aoi_id: int,
+        model_id: int,
+        status: JobStatus = JobStatus.PENDING,
     ):
-        self.status = status
-        self.created_at = created_at
-        self.aoi_id = aoi_id
-        self.model_id = model_id
         self.start_date = start_date
         self.end_date = end_date
         self.maxcc = maxcc
-        self.is_deleted = is_deleted
+        self.aoi_id = aoi_id
+        self.model_id = model_id
```

Comment:
👍

Reply:
we can keep the is_deleted here btw. Didn't remove it intentionally

Reply:
I did not see that tbh. I was just sending the "👍" because here it references the model :D